src/types/index.ts

```diff
@@ -224,15 +224,15 @@ export interface IdentityWithClaims {
 }
 
 export interface ExtrinsicData {
-  blockId?: number;
-  extrinsicIdx?: number;
+  blockId: number;
+  extrinsicIdx: number;
   address?: string | null;
```

Comment:
Why is this value still optional (or null)?

Reply:
address can be null if the user searches for system transactions. @VictorVicente can add more details about it.

Reply:
Then have it just be `string | null`. Doesn't make sense for it to be undefined as well